### DISCUSSION TITLE
Helper `validation_for` wrong with Vietnamese message

### DISCUSSION THREAD

**Discussion by datnt97** [OrchardCMS/OrchardCore#12754] [Q&A]
# THE SITUATION: 
- I'm using OrchardCore - RC2.
- I try to using `{% helper "span", validation_for: "FirstName", class:"text-error-400" %}` in Liquid. And I got an error while using `helper` with Vietnamese messages

![image](https://user-images.githubusercontent.com/59597401/199644856-64f9df98-8566-4a5f-b6aa-014608ffbac3.png)

**Comment by hishamco** [comment]
@datnt97 Orchard Core now in `v1.4`, please update your version probably there was an issue in translation (I'm not sure if it got fix). You could check Crowdin and fix the issue if still exist
